### PR TITLE
final

### PR DESCRIPTION
**Pull request by boledovicpantheon**

### CODE REVIEW

**Review by boledovicpantheon — COMMENTED** [review]
teraz som si vsimol, zabudol som odkomentovat apt-get commandy v inline

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -14,32 +14,20 @@ jobs:
...
           command: fmt
-          target: 20ubuntu/ubuntu.pkr.hcl 20ubuntu/variables.auto.pkrvars.hcl
-      
-      # format templates
-      - name: Format Template for ubuntu 22.04
-        uses: hashicorp/packer-github-actions@master
-        with:
-          command: fmt
-          target: 22ubuntu/ubuntu.pkr.hcl 22ubuntu/variables.auto.pkrvars.hcl
+          target: ubuntu2004.pkrvars.hcl ubuntu2204.pkrvars.hcl ubuntu-base.pkr.hcl
```
Comment (pmikus) [inline .github/workflows/action.yml:31]: Any reason to not include wildcards? like *.pkr.hcl *.pkrvars.hcl

**Review by pmikus — COMMENTED** [review]
Please check comments in code.

File: ubuntu-base.pkr.hcl [hunk 2]
```diff
@@ -65,23 +62,23 @@ variable "format" {
...
 }
 variable "guest_additions_mode" {
-  type    = string
+  type        = string
   description = "The method by which guest additions are made available to the guest for installation."
-  default = "upload"
+  default     = "upload"
 }
 variable "guest_additions_path" {
   type        = string
```
Comment (pmikus) [inline ubuntu-base.pkr.hcl:84]: missing description

File: ubuntu-base.pkr.hcl [hunk 3]
```diff
@@ -119,40 +116,39 @@ variable "hard_drive_interface" {
...
-  default = "8000"
+  default     = 8000
 }
 variable "iso_checksum" {
   type        = string
   description = "The checksum for the ISO file or virtual hard drive file."
   default     = ""
-  # default   = "file:http://releases.ubuntu.com/22.04/SHA256SUMS"
 }
 variable "iso_file" {
```
Comment (pmikus) [inline ubuntu-base.pkr.hcl:153]: missing description

File: ubuntu-base.pkr.hcl [hunk 4]
```diff
@@ -170,19 +166,19 @@ variable "iso_path_internal" {
...
+  default     = 1024
 }
 variable "nested_virt" {
-  type        = string
+  type        = bool
   description = "Nested virtualization: false or true."
-  default     = "true"
+  default     = true
 }
 variable "nic_type" {
```
Comment (pmikus) [inline ubuntu-base.pkr.hcl:183]: missing description

File: ubuntu-base.pkr.hcl [hunk 5]
```diff
@@ -195,76 +191,75 @@ variable "shell_scripts" {
...
+  default     = false
 }
 variable "ssh_clear_authorized_keys" {
-  type        = string
+  type        = bool
   description = "If true, Packer will attempt to remove its temporary key from ~/.ssh/authorized_keys and /root/.ssh/authorized_keys."
-  default     = "false"
+  default     = false
 }
 variable "ssh_disable_agent_forwarding" {
```
Comment (pmikus) [inline ubuntu-base.pkr.hcl:218]: missing description

File: ubuntu-base.pkr.hcl [hunk 6]
```diff
@@ -195,76 +191,75 @@ variable "shell_scripts" {
...
+  default   = "ubuntu"
   sensitive = true
 }
 variable "ssh_password_encrypted" {
-  type        = string
-  default     = "$6$exDY1mhS4KUYCE/2$zmn9ToZwTKLhCw.b4/b.ZRTIZM30JZ4QrOQ2aOXJ8yk96xpcCof0kxKwuX1kqLG/ygbJ1f8wxED22bTL4F46P0"
+  type    = string
+  default = "$6$exDY1mhS4KUYCE/2$zmn9ToZwTKLhCw.b4/b.ZRTIZM30JZ4QrOQ2aOXJ8yk96xpcCof0kxKwuX1kqLG/ygbJ1f8wxED22bTL4F46P0"
 }
 variable "ssh_port" {
```
Comment (pmikus) [inline ubuntu-base.pkr.hcl:246]: description

**Review by pmikus — COMMENTED** [review]
Few comments, otherwise good

**Comment by boledovicpantheon** [comment]
@pmikus mozem to close-nut?

**Comment by pmikus** [comment]
Please use english everywhere in comments, commits or messages.
Yes you can merge

**Pull request merged**